deployments/types/rialto.json

```diff
@@ -10,5 +11,8 @@
 		"_enum": {
 			"RialtoToMillauConversionRate": "u128"
 		}
+	},
+	"ValidationCodeHash": {
+		"Hash": "Hash"
 	}
```

Comment:
```suggestion
	"ValidationCodeHash": "H256",
```

Reply:
Addressed on here and generated new `types-rialto.json` with this change.

Reply:
This should be exactly as I've suggested. The type does not have "Hash" field, but rather is just a wrapper type around `H256`.

Reply:
sorry! i see the difference now. Changing it.

Reply:
solved.